src/aat/resources/features/F-140 - Get Linked Case V2 External/F-140.feature

```diff
@@ -0,0 +1,112 @@
+#====================================
+@F-140
+Feature: F-140: Retrieve a case by case reference
+#====================================
+
+Background:
+    Given an appropriate test context as detailed in the test data source
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-140.1
+Scenario: should get 400 when case reference invalid
+
+    Given a user with [an active profile in CCD],
+
+     When a request is prepared with appropriate values,
+      And the request [contains an invalid case reference],
+      And it is submitted to call the [retrieve a linked case by case reference] operation of [CCD Data Store],
+
+     Then a negative response is received,
+      And the response [contains an HTTP-400 Bad Request],
+      And the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-140.2
+Scenario: should get 404 when case reference does not exist
+
+    Given a user with [an active profile in CCD],
+
+     When a request is prepared with appropriate values,
+      And the request [contains a case reference that does not exist],
+      And it is submitted to call the [retrieve a linked case by case reference] operation of [CCD Data Store],
+
+     Then a negative response is received,
+      And the response [contains an HTTP-404 Not Found],
+      And the response has all other details as expected.
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-140.3
+Scenario: must return 404 when request provides authentic credentials without authorised access to the operation
+
+    Given a case that has just been created as in [Unauthorised_Case_Creation_Data],
```

Comment:
Switch to [Standard_Full_Case_Creation_Data] as with other tests but use an unauthorised user in the call to endpoint, i.e. override `users.invokingUser` with one from another jurisdiction in S.140.3.

Reply:
[54f1d4f](https://github.com/hmcts/ccd-data-store-api/pull/1920/commits/54f1d4fcf3faac512d1a6278a71e82b99881cce4)